### PR TITLE
slicer-uploads fix

### PR DESCRIPTION
**Pull request by vvchik**
starting from SuperSlicer and PrusaSlicer 2.3.0 location of this config changed.

### CODE REVIEW

File: docs/setup/slicer-uploads.md [hunk 1]
```diff
@@ -2,10 +2,13 @@
 
 You can upload your sliced `gcode` from within PrusaSlicer or SuperSlicer. 
 
-- With PrusaSlicer / SuperSlicer open, click through to `Printer Settings -> General` and find the `Printer Host Upload` section.
+- With PrusaSlicer / SuperSlicer open, click the "cog" icon right of the Printer profiles combo box and select `Add physical printer`
+- Type a descriptive printer name
+- chose proper printer preset
```
Comment (cadriel) [inline docs/setup/slicer-uploads.md:7]: `chose` -> `choose a printer preset`

**Review by cadriel — COMMENTED** [review]
Otherwise looks good.

**Comment by vvchik** [comment]
fixed

**Comment by cadriel** [comment]
I had to rebase and squash this locally, but it's now merged. Thanks!

**Pull request closed without merging**